README.md

```diff
@@ -16,7 +16,8 @@ CleanRL is a Deep Reinforcement Learning library that provides high-quality sing
 
 
 * 📜 Single-file implementation
-   * *Every detail about an algorithm is put into the algorithm's own file.* It is therefore easier to fully understand an algorithm and do research with.
+   * *Every detail about an algorithm variant is put into a single standalone file.* 
+   * For example, our `ppo_atari.py` only has 340 lines of code but contains all implementation details on how PPO works with Atari games, so it is a great reference implementation to read for folks who do not wish to read an entire modular library.
```

Comment:
“reference implementation” is already mentioned here. The reason I am hesitant to put it in the title is because our paper already has a title that does not have the “reference” word in it, so I’d avoid changing it.